src/components/UI/docs/Code.tsx

```diff
@@ -0,0 +1,46 @@
+// Libraries
+import { Code as ChakraCode, Stack, Text } from '@chakra-ui/react';
+import { FC } from 'react';
+
+interface Props {
+  code: any;
+}
+
+export const Code: FC<Props> = ({ code }) => {
+  return (
+    !!code.inline ?
+      (
+        <Text
+          as='span'
+          background='code-bg'
+          fontFamily='"JetBrains Mono", monospace'
+          fontWeight={400}
+          fontSize='md'
+          lineHeight={4}
+          letterSpacing='1%'
+          pb={2}
+          mb={-2}
+        >
+          {code.children[0]}
+        </Text>
+      )
+    : 
+      (
+        <Stack>
+          <ChakraCode
+            overflow='hidden'
+            p={6}
+            background='code-bg-contrast'
+            color='green.50'
+            fontFamily='"JetBrains Mono", monospace'
```

Comment:
`fontFamily`, `fontWeight`, `fontSize`, `lineHeight`, `letterSpacing` and `color` should be defined on theme's textStyles.

Reply:
Everything but `color` was able to go into the textStyles. For some reason I couldn't get `color` working in there.

Reply:
I could add a `Text` in here, and add some padding on the right if needed I think though if that's preferred.

Reply:
@corwintines I think using `Code` makes sense, we can padding with a `Box` later if needed